### ISSUE TITLE
Do you have any plan to upgrade this and migrate to swift 4??

### ISSUE THREAD

**Issue by ktjd123** [pj4533/wat#1]
I think this is good project.

It would be very happy if you migrate this to swift version 4.

Thank you and appreciate your work

**Comment by ktjd123** [comment]
@pj4533

**Comment by pj4533** [comment]
Eh...it was a little side project/experiment I did years ago, not really actively working on it.  I’ll leave this issue open though, in case others want to build on it.   Feel free to run with it!

**Comment by ktjd123** [comment]
Thank you for reply!! Actually I tried to migrate it. But it was too old version for me 😭😭 (I learned swift from version 3 ). Maybe I can learn older version of swift at this chance. :)

**Comment by pj4533** [comment]
Yeah there is some weirdo swift in there due to the need for the ObjC pointer stuff...that might have all changed in the later versions of Swift, I am not sure.
